.travis.yml

```diff
@@ -10,11 +10,29 @@ os:
 
 install:
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install make; fi
-- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install make; fi
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    sudo apt-get install make;
+    sudo apt-get update -qq;
+    sudo apt-get install -qq -y software-properties-common uidmap;
+    sudo add-apt-repository -y ppa:projectatomic/ppa;
+    sudo apt-get update -qq;
+    sudo apt-get -qq -y install podman;
+  fi
 
-script:
-- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then mingw32-make build; fi
-- if [[ "$TRAVIS_OS_NAME" != "windows" ]]; then make unit build; fi
+script: |
+  if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
```

Comment:
Since this seems like technical debt to actually enable these tests in multiple environments, can we just run the tests in only linux to start and create issues to get them enabled everywhere else at a later time? Even if we don't remove these steps for now, I think it would be valuable to track it somewhere.

Reply:
for sure